build/copy-jsbeautify.js

```diff
@@ -16,6 +16,14 @@ function copy(from, to) {
 	}
 }
 
+if (!fs.existsSync(path.join(__dirname, '..', 'lib', 'umd'))) {
```

Comment:
This isn't needed to fix tests, but included to mirror the fix for `esm`

Reply:
This should be fixed by e6deb0b91579ee9e6d34893f5317559c22ba5710

Reply:
Awesome! I'll update my PR!